mxcubeweb/core/components/harvester.py

```diff
@@ -0,0 +1,505 @@
+                    "acronym": "cryoprotectant",
+                    "img_url": "https://htxlab.embl.org//rawimages/2021//CD032401/6/FORMULATRIX_CD032401_6_08-09-2021_06_04_01_00_99_Vis.jpg",
+                    "img_target_x": 54.466230936819,
+                    "img_target_y": 58.026175213675,
+                },
+                {
+                    "crystal_uuid": "5d5dfc88-009d-4486-8526-33bf58c4d5d9",
+                    "name": "TEST13",
+                    "state": "failed",
+                    "acronym": "cryoprotectant",
+                    "img_url": "https://htxlab.embl.org//rawimages/2021//CD032401/6/FORMULATRIX_CD032401_6_08-09-2021_06_02_01_00_99_Vis.jpg",
+                    "img_target_x": 32.710943173566,
+                    "img_target_y": 51.110576923077,
+                },
+                {
+                    "crystal_uuid": "46aca4a4-0c00-4685-b1e1-53071bdcb66d",
+                    "name": "TEST4",
+                    "state": "harvested",
+                    "acronym": "cryoprotectant",
+                    "img_url": "https://htxlab.embl.org//rawimages/2021//CD032401/6/FORMULATRIX_CD032401_6_08-09-2021_06_04_01_00_99_Vis.jpg",
+                    "img_target_x": 38.632400075415,
+                    "img_target_y": 41.762286324786,
+                },
+            ]
+
+        return crystal_list
+
+    def get_global_state(self):
+        try:
+            return HWR.beamline.harvester_maintenance.get_global_state()
+        except Exception:
+            return "OFFLINE", "OFFLINE", "OFFLINE"
+
+    def send_data_collection_info_to_crims(self) -> bool:
+        """Send Data collected to CRIMS
+
+        Return (bool): Whether the request failed (false) or not (true)
+        """
+        dataCollectionGroupId = ""
+        crystal_uuid = ""
+        harvester_device = HWR.beamline.harvester
+
+        try:
+            rest_token = HWR.beamline.lims.lims_rest.get_rest_token()
+            proposal = HWR.beamline.session.get_proposal()
+
+            crims_url = harvester_device.crims_upload_url
+
+            queue_entries = HWR.beamline.queue_model.get_all_dc_queue_entries()
+            dc_id = ""
+            for qe in queue_entries:
+                dataCollectionGroupId = qe.get_data_model().lims_group_id
+                crystal_uuid = (
+                    qe.get_data_model().get_sample_node().crystals[0].crystal_uuid
+                )
+                dc_id = qe.get_data_model().id
+
+                Crims.send_data_collection_info_to_crims(
+                    crims_url,
+                    crystal_uuid,
+                    dataCollectionGroupId,
+                    dc_id,
+                    proposal,
+                    rest_token,
+                )
+            return True
+        except Exception as ex:
+            msg = "get all queue entries failed, reason:  %s" % str(ex)
+            logging.getLogger("user_level_log").exception(msg)
+            return False
```

Comment:
There should be no need to have the exception in the message if it is the `exception` method of the logger that is used. This should be just fine:

```suggestion
        except Exception as ex:
            msg = "get all queue entries failed, reason"
            logging.getLogger("user_level_log").exception(msg)
            return False
```